org.skytemple.SkyTemple.yml

```
@@ -0,0 +1,144 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
```

this should just be the command name
```suggestion
command: run.sh
```